messages/en.json

```diff
@@ -614,14 +640,14 @@
     "invalid_permissions": "You do not have permission to use that command.",
     "player_joined_deadchat": "\u0002{0}\u0002 has joined the deadchat.",
     "multiple_joined_deadchat": "\u0002{0}\u0002 and \u0002{1}\u0002 have joined the deadchat.",
-    "joined_deadchat": "You are now in the deadchat.",
+    "joined_deadchat": "You are now in the deadchat. You can leave by sending !leave and disable automatic joining on death using !deadchat.",
     "list_deadchat": "Players: {0}",
     "force_leave_deadchat": "You have been forcibly removed from the deadchat by \u0002{0}\u0002.",
     "player_force_leave_deadchat": "\u0002{0}\u0002 has been removed from the deadchat by \u0002{1}\u0002.",
     "leave_deadchat": "You have left the deadchat.",
     "player_left_deadchat": "\u0002{0}\u0002 has left the deadchat.",
     "chat_on_death": "You will now join the deadchat on death.",
-    "no_chat_on_death": "You will no longer join the deadchat on death.",
+    "no_chat_on_death": "You will no longer join the deadchat on death. You can still manually join by sending !join in a private message.",
```

Comment:
Don't hardcode !, command prefix may be different. Needs clarification that such is to be used after the game has already started